SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/LayeredLBStrategy.cpp

```diff
@@ -17,6 +17,29 @@ ULayeredLBStrategy::ULayeredLBStrategy()
 {
 }
 
+FString ULayeredLBStrategy::ToString() const
+{
+	const FName LocalLayerName = GetLocalLayerName();
+	const UAbstractLBStrategy* const* LBStrategy = LayerNameToLBStrategy.Find(LocalLayerName);
+
+	FString Description =
+		FString::Printf(TEXT("Layered, LocalLayerName = %s, LocalVirtualWorkerId = %d, LayerStrategy = %s"), *LocalLayerName.ToString(),
+						LocalVirtualWorkerId, *LBStrategy ? *(*LBStrategy)->ToString() : TEXT("NoStrategy"));
+
+	if (VirtualWorkerIdToLayerName.Num() > 0)
```

Comment:
Can I suggest adding a comment with proposed outcome or an example of the Description string?

Reply:
I prefer not too, since it should be easy to locate in the logs with the information shown here, plus it stops the example accidentally becoming out of date.

Reply:
If I were to locate this in the logs my first port of call would be this function and I'd prefer to have an example as superfast shorthand rather than working it out by inferring it from the code. Additional the comment would spell out the intention of the code, which can be checked against the output in case things are missing or misunderstood. 
The out of date thing is a fair point, but I def err on the side of having more comments and relying on PR's to keep things in sync, which shouldn't be too hard since the code is all one local block. In the end its personal preference I guess.